BUILD

```diff
@@ -280,6 +280,21 @@ grpc_cc_library(
     ],
 )
 
+grpc_cc_library(
```

Comment:
this will break down in the internal google build IIC, we probably want to keep referring to nanopb as an external dep in `BUILD`, but have it refer to this new rule inside `bazel/grpc_build_system.bzl`

Reply:
Thanks Alex for  the heads-up. Let me run the import script on this PR first to see if it works, and will go back and try your approach if it does not.